target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8065.dtsi

```diff
@@ -49,68 +55,53 @@
 	/delete-node/opp-1200000000;
 
 	opp-384000000 {
-		opp-hz = /bits/ 64 <384000000>;
 		opp-microvolt-speed0-pvs0-v0 = <975000>;
 		opp-microvolt-speed0-pvs1-v0 = <950000>;
 		opp-microvolt-speed0-pvs2-v0 = <925000>;
 		opp-microvolt-speed0-pvs3-v0 = <900000>;
 		opp-microvolt-speed0-pvs4-v0 = <875000>;
 		opp-microvolt-speed0-pvs5-v0 = <825000>;
 		opp-microvolt-speed0-pvs6-v0 = <775000>;
-		opp-supported-hw = <0x1>;
-		clock-latency-ns = <100000>;
 	};
 
 	opp-600000000 {
-		opp-hz = /bits/ 64 <600000000>;
 		opp-microvolt-speed0-pvs0-v0 = <1000000>;
 		opp-microvolt-speed0-pvs1-v0 = <975000>;
 		opp-microvolt-speed0-pvs2-v0 = <950000>;
 		opp-microvolt-speed0-pvs3-v0 = <925000>;
 		opp-microvolt-speed0-pvs4-v0 = <900000>;
 		opp-microvolt-speed0-pvs5-v0 = <850000>;
 		opp-microvolt-speed0-pvs6-v0 = <800000>;
-		opp-supported-hw = <0x1>;
-		clock-latency-ns = <100000>;
 	};
 
 	opp-800000000 {
-		opp-hz = /bits/ 64 <800000000>;
 		opp-microvolt-speed0-pvs0-v0 = <1050000>;
 		opp-microvolt-speed0-pvs1-v0 = <1025000>;
 		opp-microvolt-speed0-pvs2-v0 = <1000000>;
 		opp-microvolt-speed0-pvs3-v0 = <975000>;
 		opp-microvolt-speed0-pvs4-v0 = <950000>;
 		opp-microvolt-speed0-pvs5-v0 = <900000>;
 		opp-microvolt-speed0-pvs6-v0 = <850000>;
-		opp-supported-hw = <0x1>;
-		clock-latency-ns = <100000>;
 	};
 
 	opp-1000000000 {
-		opp-hz = /bits/ 64 <1000000000>;
 		opp-microvolt-speed0-pvs0-v0 = <1100000>;
 		opp-microvolt-speed0-pvs1-v0 = <1075000>;
 		opp-microvolt-speed0-pvs2-v0 = <1050000>;
 		opp-microvolt-speed0-pvs3-v0 = <1025000>;
 		opp-microvolt-speed0-pvs4-v0 = <1000000>;
 		opp-microvolt-speed0-pvs5-v0 = <950000>;
 		opp-microvolt-speed0-pvs6-v0 = <900000>;
-		opp-supported-hw = <0x1>;
-		clock-latency-ns = <100000>;
 	};
 
 	opp-1400000000 {
-		opp-hz = /bits/ 64 <1400000000>;
```

Comment:
I can't really find where this is reproduced... This is needed to make the scaling works 
Same for opp-supported and clock

Reply:
These definitions are already in the ipq8064.dtsi

Since the node name is the same, we are referencing these very blocks and are just _updating_ them.
That's also why we need to use delete-node above.

https://github.com/openwrt/openwrt/blob/master/target/linux/ipq806x/files-5.4/arch/arm/boot/dts/qcom-ipq8064.dtsi#L145

Reply:
you are right. sorry